README.md

```diff
@@ -85,18 +85,18 @@ nmcli device set eth0 managed no
     ```bash
     docker run -v $OVPN_DATA:/etc/openvpn --rm salvoxia/openvpn-tap \
       ovpn_genconfig \
-          -u udp://VPN.SERVERNAME.COM:PORT \
-          -t \
-          -B \
-          --bridge-name 'br0' \
-          --bridge-eth-if 'eth0' \
-          --bridge-eth-ip '192.168.0.199' \
-          --bridge-eth-subnet '255.255.255.0' \
-          --bridge-eth-broadcast '192.168.0.255' \
-          --bridge-eth-gateway '192.168.0.1' \
-          --bridge-eth-mac 'b8:32:ac:8b:17:2e' \
-          --bridge-dhcp-start '192.168.0.200' \
-          --bridge-dhcp-end '192.168.0.220'
+          -u udp://VPN.SERVERNAME.COM:PORT \  # Your public IP/domain and port forwarded to Docker
+          -t \                                 # Enable TAP mode
+          -B \                                # Enable bridging mode
+          --bridge-name 'br0' \               # Bridge interface name (Optional, change if br0 already exists)
```

Comment:
The parameter itself is actually not optional, but `br0` is a good default value unless another bridge already exists.

Reply:
THANKS, that is my misunderstanding